client/components/Layout/Layout.tsx

```diff
@@ -29,7 +29,12 @@ const Layout = (props: Props) => {
 		if (block.type === 'pubs') {
 			return (
 				<div className="layout-pubs-block">
-					<LayoutPubs key={index} content={block.content} pubs={pubBlocksLists[index]} />
+					<LayoutPubs
+						key={index}
```

Comment:
Does this `key` prop need to be on the parent div instead?

Reply:
oh, my god, yes! I kept getting key errors and I couldn't for the life of me figure out why because I missed the div. Thanks.